src/tsdb/m3.act

```diff
@@ -17,30 +17,31 @@ import net
 import json
 import time
 import logging
+from tsdb.common import Metric
 import tsdb.http
 
-class Metric(object):
-    @property
-    name: str
-    @property
-    tags: dict[str, str]
-    @property
-    value: float
-    @property
-    timestamp: int
-
-    def __init__(self, name: str, tags: dict[str, str], value: float, timestamp: int):
-        self.name = name
-        self.tags = tags
-        self.value = value
-        self.timestamp = timestamp
-
-    def __str__(self):
-        # XXX including str(self.timestamp) in this repr triggers a segfault for reasons unknown
-        return "<tsdb.Metric" + " " + str(self.name) + " " + str(self.value) + " " + str(self.tags) + " " + str(self.timestamp) + ">"
-
-    def __repr__(self):
-        return self.__str__()
+#class Metric(object):
```

Comment:
Can we just remove this instead of commenting out? we always have access to it again through git